pycqed/measurement/calibration_points.py

```diff
@@ -72,7 +75,7 @@ def get_states(self, qb_names=None):
         return {qbn: [s[self.qb_names.index(qbn)] for s in self.states]
                 for qbn in qb_names}
 
-    def get_indices(self, qb_names=None):
+    def get_indices(self, qb_names=None, prep_params=None):
```

Comment:
a brief comment woul be useful, e.g. 
> prep_params as returned by `QuDev_Transmon.prep_params`

Reply:
Brief and not-so-brief comments are always useful :p 
I added it now, thanks :)